app/products/calls/connection/connection.ts

```diff
@@ -206,30 +201,45 @@ export async function newConnection(
         InCallManager.start({media: 'audio'});
         InCallManager.stopProximitySensor();
 
-        peer = new Peer(null, iceConfigs);
-        peer.on('signal', (data: any) => {
-            if (data.type === 'offer' || data.type === 'answer') {
-                ws.send('sdp', {
-                    data: deflate(JSON.stringify(data)),
-                }, true);
-            } else if (data.type === 'candidate') {
-                ws.send('ice', {
-                    data: JSON.stringify(data.candidate),
-                });
+        peer = new RTCPeer({iceServers: iceConfigs || []}, () => InCallManager.stop());
```

Comment:
I believe this was because we had to be absolutely sure the peer and tracks had been removed before calling stop, but it looks like RTCPeer simplifies this, so you're right -- no need for the callback. Removed.  
Working on the merge conflicts now.